cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -418,3 +420,40 @@ instance Crypto.Crypto crypto => ToJSON (VMap VB VB (Shelley.KeyHash    'Shelley
 
 instance Crypto.Crypto crypto => ToJSON (VMap VB VP (Shelley.Credential 'Shelley.Staking   crypto) (Shelley.CompactForm Shelley.Coin)) where
   toJSON = toJSON . fmap fromCompact . VMap.toMap
+
+-----
+
+instance ToJSON (Consensus.StakeSnapshot crypto) where
+  toJSON
+    Consensus.StakeSnapshot
+    { Consensus.sMarkPool
+    , Consensus.sSetPool
+    , Consensus.sGoPool
+    , Consensus.sMarkTotal
+    , Consensus.sSetTotal
+    , Consensus.sGoTotal
+    } = object
+    [ "poolStakeMark" .= sMarkPool
```

Comment:
Can we refactor the key pairs as you did in 4205?

Reply:
Done!